### PR TITLE
[#145964569] Refresh dashboard

### PR DESCRIPTION
**Pull request by waggles**
Depends on #1069  , see https://github.com/tablexi/nucore-open/compare/public-dashboard-145964569...refresh-dashboard-145964569
Phase 2 of https://www.pivotaltracker.com/story/show/145964569

Adds auto-refresh/show last refreshed at

### CODE REVIEW

**Comment by waggles** [comment]
also updated~

File: vendor/engines/secure_rooms/app/controllers/secure_rooms/occupancies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module SecureRooms
+
+  class OccupanciesController < ApplicationController
+
+    layout "plain"
+
+    before_action :load_occupancies
+
+    def index
+      @refresh_url = refresh_facility_secure_room_occupancies_url(current_facility, @secure_room.dashboard_token)
```
Comment (jhanggi) [inline vendor/engines/secure_rooms/app/controllers/secure_rooms/occupancies_controller.rb]: Is there a way to collapse these two actions into one? Do something like `render ..., layout: !request.xhr?` so it has no layout when being fetched with ajax.

File: vendor/engines/secure_rooms/app/controllers/secure_rooms/occupancies_controller.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    end
+
+    def refresh
+      render partial: "secure_rooms/shared/secure_rooms_dashboard", locals: { room: @secure_room }
+    end
+
+    private
+
+    def load_occupancies
```
Comment (jhanggi) [inline vendor/engines/secure_rooms/app/controllers/secure_rooms/occupancies_controller.rb]: Would `load_secure_room` be a little closer to what this is actually doing?

File: vendor/engines/secure_rooms/app/controllers/secure_rooms/occupancies_controller.rb (hunk 1 above)
Comment (waggles) [inline vendor/engines/secure_rooms/app/controllers/secure_rooms/occupancies_controller.rb]: hmm I would have to move the javascript either into the layout or into assets somehow... I'm not sure which setup I prefer

File: vendor/engines/secure_rooms/app/controllers/secure_rooms/occupancies_controller.rb (hunk 2 above)
Comment (waggles) [inline vendor/engines/secure_rooms/app/controllers/secure_rooms/occupancies_controller.rb]: yeah good point, updating

**Pull request merged**